frontend/Search.vue

```diff
@@ -35,102 +35,147 @@
+                                    v-for="i in ($STRINGS.MODE_COUNT - 0)"
+                                    :key="i"
+                                    :value="$STRINGS['MODE_KEY_' + i]"
+                                    :label="$STRINGS['MODE_TITLE_' + i]">
+                            </v-radio>
+                        </v-radio-group>
 
-            <TaxonomyAutocomplete v-model="taxFilter"></TaxonomyAutocomplete>
+                        <TaxonomyAutocomplete v-model="taxFilter"></TaxonomyAutocomplete>
 
-            <v-tooltip v-if="!$ELECTRON && !hideEmail" open-delay="300" top>
-                <template v-slot:activator="{ on }">
-                    <v-text-field v-on="on" label="Notification Email (Optional)" placeholder="you@example.org" v-model="email"></v-text-field>
-                </template>
-                <span>Send an email when the job is done.</span>
-            </v-tooltip>
-        </div>
-        </panel>
-    </v-flex>
-    <v-flex>
-        <panel>
-        <template slot="content">
-            <div class="actions" :style="!$vuetify.breakpoint.xsOnly ?'display:flex; align-items: center;' : null">
-            <v-item-group class="v-btn-toggle">
-                <v-btn color="primary" :block="false" x-large v-on:click="search" :disabled="searchDisabled"><v-icon>{{ $MDI.Magnify }}</v-icon>&nbsp;Search</v-btn>
-                <v-btn v-if="isMultimer" color="secondary" :block="false" x-large v-on:click="goToMultimer"><v-icon>{{ $MDI.Multimer }}</v-icon>&nbsp;Go to Multimer</v-btn>
-            </v-item-group>
-            <div :style="!$vuetify.breakpoint.xsOnly ? 'margin-left: 1em;' : 'margin-top: 1em;'">
-                <span><strong>Summary</strong></span><br>
-                Search <template v-if="taxFilter">
-                    <strong>{{ taxFilter.text }}</strong> in
-                </template>
-                <template v-if="database.length == databases.length">
-                    <strong>all available</strong> databases
+                        <v-divider class="my-2"></v-divider>
+
+                        <v-radio-group v-model="iterativeSearch">
+                            <v-tooltip open-delay="300" top>
+                                <template v-slot:activator="{ on }">
+                                    <label v-on="on">
+                                        Iterative search
+                                        <v-icon color="#FFFFFFB3" style="margin-top:-3px" small v-on="on">{{ $MDI.HelpCircleOutline }}</v-icon>
+                                    </label>
+                                </template>
+                                <span>Improve sensitivity of search by performing an iterative search (--num-iterations 3).</span>
+                            </v-tooltip>
+
+                            <v-radio label="On" :value="true"></v-radio>
+                            <v-radio label="Off" :value="false"></v-radio>
+                        </v-radio-group>
```

Comment:
Please keep email in addition to the new iterative search

Reply:
done